crates/swc_ecma_transforms_compat/src/es2015/block_scoping/mod.rs

```diff
@@ -717,8 +689,8 @@ impl Visit for InfectionFinder<'_> {
 
 #[derive(Debug)]
 struct FlowHelper<'a> {
-    has_continue: bool,
-    has_break: bool,
+    continue_stmts: AHashSet<Option<Ident>>,
```

Comment:
I think we should use `AHashSet<Option<JsWord>>` instead.
This will error if there are multiple `continue foo`